src/components/ComponentIndexPage/index.js

```diff
@@ -20,21 +23,62 @@ function ComponentIndexPage() {
     ALL_COMPONENTS_QUERY
   );
 
+  const href = '/get-started/about-carbon';
+
+  const items = [];
+
   return (
-    <dl>
-      {components.edges.map(({ node }) => {
-        const { name, description, maintainer } = node;
-        const key = `${name}:${maintainer}`;
-        return (
-          <React.Fragment key={key}>
-            <dt>
-              {maintainer}: {name}
-            </dt>
-            {description ? <dd>{description}</dd> : null}
-          </React.Fragment>
-        );
-      })}
-    </dl>
+    <>
+      <Search
+        className="component-index-search"
+        id="search-1"
+        placeHolderText="Search"
+      />
+      <Dropdown
+        ariaLabel="Dropdown"
+        id="carbon-dropdown-example"
+        items={items}
+        label="Sort by A to Z"
+        className="placeholder-sort"
+      />
+
+      <dl>
```

Comment:
Might help to clean-up the markup here a bit from the demo, maybe something like:

- `<dl>` goes to `<section aria-label="Component index">`?
- Each card ends up being its own "row" (can use the component from CCR)
- Each card is an `article`
  - The `header` of the article is the component name
  - The body of the article is the description
  - The `footer` of the article contains the links

for example:

```html
<section aria-label="Component index">
  <article class="component-index-item">
    <div class="component-index-item__image">
      <img src="..." alt="...">
    </div>

    <div class="component-index-item__content">
      <header class="component-index-item__name">
        Component name
      </header>

      <p class="component-index-item__description">
        Lorem, ipsum dolor sit amet consectetur adipisicing elit.
      </p>
      
      <footer class="component-index-item__info">
        <a class="component-index-item__link" href="">Link 1</a>
        <a class="component-index-item__link" href="">Link 1</a>
      </footer>
    </div>
  </article>
</section>
```
Curious what you think 👀

Reply:
It might also help to use the `Row` and `Column` components, as well, from `carbon-components-react`. Let me know if I can help out with using those at all!

Reply:
Yes this all looks awesome! let me play around with the `Row` & `Column` components as well